<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="LEDShining">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>LED Shining</string>
    </property>
    <property name="shortcut">
     <string>A</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LED2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LED2</string>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LED3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LED3</string>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LED4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LED4</string>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LED1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>LED1</string>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LEDSwitchingOn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Switching on</string>
    </property>
    <property name="shortcut">
     <string>S</string>
    </property>
   </widget>
   <widget class="QLabel" name="LED1_pic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LED2_pic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LED3_pic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LED4_pic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>48</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Switching Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LEDSwitchingOff">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Switching off</string>
    </property>
    <property name="shortcut">
     <string>D</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
